tools/src/publish/bin/bump-package.ts

```diff
@@ -26,15 +26,19 @@ try {
   if (version != null) {
     if (packageJson.version === version) {
       console.info(`Package version is already set to ${version}.`);
-      process.exit(0);
+      // process.exit(0);
```

Comment:
Is this temporarily commented out, or should it be removed for good?

Reply:
removed it. I now update the package.json always to be able to mark it.